src/msw/dc.cpp

```diff
@@ -50,28 +50,9 @@
 #include "wx/private/textmeasure.h"
 
 #ifdef _MSC_VER
-    // In the previous versions of wxWidgets, AlphaBlend() was dynamically
-    // loaded from msimg32.dll during the run-time, so we didn't need to link
-    // with this library. Now that we use the function statically, we do need
-    // to link with it and we do it implicitly from here for MSVC users to
-    // avoid breaking the build of the existing projects which didn't link with
-    // this library before.
     #pragma comment(lib, "msimg32")
-
-    // Indicate that we should just use the functions from gdi32.dll and
-    // msimg32.dll directly.
-    #define USE_STATIC_GDI_FUNCS
-#else // !_MSC_VER
-    // In other compilers, e.g. MinGW, we don't have anything similar to
-    // #pragma comment(lib) used above, so we continue loading AlphaBlend()
-    // dynamically, if possible.
-    //
-    // We also load some GDI functions not present in MinGW libraries
-    // dynamically.
-    #include "wx/dynlib.h"
-
-    #define USE_DYNAMIC_GDI_FUNCS
-#endif // _MSC_VER/!_MSC_VER
+    #pragma comment(lib, "gdi32")
```

Comment:
Very minor, but I don't think we ever need this, as we always link with this one anyhow.

Reply:
There might be some scenarios where it doesn't have gdi32.
For example when building static libraries. And then you use CMake's own FindwxWidgets to find these libraries and link with them.  It doesn't have `gid32` in the win32 [library list](https://gitlab.kitware.com/cmake/cmake/-/blob/master/Modules/FindwxWidgets.cmake#L470). So without this pragma, the final executable won't know it needs gid32.

There is `include/msvc/wx/setup.h` but I think this is only used if you manually add `include/msvc` as the first include path?

Reply:
Sorry, I don't understand how could there be any scenarios in which gdi32 is not linked — if this were possible, we'd be getting link errors already because we use plenty of functions from it non-dynamically. I think CMake might be linking Windows GUI programs with it by default, as this would make sense.

Reply:
Ok, I've removed it.